### DISCUSSION TITLE
Not able to mint NFT of any lesson

### DISCUSSION THREAD

**Discussion by 0xtdey** [Cyfrin/foundry-full-course-cu#207] [Q&A] [answered]
I have completed till lesson number 6 but I am not able to mint any NFT even after solving the challenge. Everytime I press write button on sepolia etherscan it shows like the picture attached, in metamask and the transaction fails. 
<img width="242" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91402167/8b801ff4-2304-4a2a-80d1-d63d6e6f67b5">

**Comment by ChefAharoni** [comment]
This is a general error in solidity when there is something wrong in the contract.

In the challenges this error is given when the answer is wrong.

**Reply by 0xtdey** [reply]
Have that been the case then, I would have been able to mint the first lesson NFT, where we just need to put the Twitter handle and write the contract.

**Comment by alymurtazamemon** [comment]
@tamoghna-dey This means you did not solve the challenge yet.

**Reply by 0xtdey** [reply]
I did solve the challenge, for lesson 1 we just need to put our twitter handle and write the contract, for lesson 2 we need to put 123 in place of uint256 and our twitter handle, still how come is it wrong?

**Reply by alymurtazamemon** [reply]
@tamoghna-dey Is it a lesson 2 challenge? And are you using this address for the first time to mint this nft?

**Reply by 0xtdey** [reply]
This is a lesson 2 challenge - 0x5c1ddb86F11BB46D3067C702AC554aEaED9ff8f0 
I am minting it on sepolia, I have tried multiple times to mint it, with the same address but the problem remains same

**Reply by alymurtazamemon** [reply]
@tamoghna-dey Working fine for me!

<img width="857" alt="Screenshot 2023-07-01 at 9 12 15 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/56123405/8de5161c-7224-4c08-b05e-b086e093c82f">

**Reply by 0xtdey** [reply, marked as answer]
Yaa, I made a new account on Metamask and then tried to mint, then it worked. Still don't know what the exact issue is.
@alymurtazamemon , @ChefAharoni thanks for your response guys!
